Makefile

```diff
@@ -0,0 +1,44 @@
+# Define the dependencies
+DEPENDENCIES = chafa convert jq curl
+
+# Define the installation directories
+INSTALL_DIR = /usr/local
+
+# Installation instructions
+install: check_dependencies
+	@echo "\033[1;32m Installing... \033[0m"
+	sudo mkdir -pv $(INSTALL_DIR)
+	sudo cp -rv bin $(INSTALL_DIR)
+	sudo cp -rv share $(INSTALL_DIR)
+	@echo "Installation complete."
+	@echo "updating your $PATH variable"
+
+	@echo -e "\033[1;32m DONE \033[0m pokeshell is now installed"
+	@echo -e "\033[1;32m RELOAD YOUR SHELL BEFORE USING POKESHELL \033[0m"
+
+# Uninstallation instructions
+uninstall:
+	@echo "Uninstalling..."
+
+	@echo "Removing files"	 
+	sudo rm -rf $(INSTALL_DIR)/bin/pokeshell
+	@echo "Removing Complitions"
+	sudo rm -rf $(INSTALL_DIR)/share/bash-completion/completions/pokeshell
```

Comment:
the `-r` flag is doing nothing here as these are files
the `-f` flag is not needed anyway either I think, can you explain where the current invocation will fail (just `rm`) compared to `rm -f`?
Also I'd like to keep the `-v` flag for being transparent with users. Shellscripts shouldn't be trusted willy-nilly from the internet and `-v` makes what we are removing on their filesystem clear.